HelpSource/scdoc.css

```diff
@@ -492,14 +502,14 @@ td p {
 
 .doclink {
     font-size: 9pt;
-    color: #888;
+    color: var(--sc-neutral-darker);
     text-align: right;
     margin-top: 2em;
     margin-bottom: 2em;
-    border-top: 1px solid #ddd;
+    border-top: 1px solid var(--sc-neutral-lighter);
 }
 .doclink a {
-    color: #444;
+    color: var(--sc-neutral-darker);
```

Comment:
should we give two different shades to `.doclink` and `.docklink a`? would be great to keep the differentiation

Reply:
yes, do you have a preference which color to pick?

Reply:
just two different shades of the main text color, for example (didn't test it though):
```css
.doclink { color: var(--sc-neutral-darker) }
.doclinka { color: var(--sc-text-color-lighter) }
```